games/static/games/proto.js

```diff
@@ -321,6 +353,8 @@ class Board {
   moveAliens() {
     this.determineDirection();
     this.alienLocations = this.calculateAlienLocations();
+    this.updateCurrentTotalAlienHealth()
```

Comment:
i'm not sure its the place for calculating the score
is the score effected only from the hit?
idea: when a rocket hits an alien update score

Reply:
it does not work so well...